pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/EgenerklaeringPleieOgOmsorgsarbeid.kt

```diff
@@ -27,6 +33,24 @@ val egenerklaeringPleieOgOmsorgsarbeid = createAttachment<LangBokmalNynorskEngli
     ),
     includeSakspart = true
 ) {
+    vedlegg(returadresse) { aarEgenerklaringOmsorgspoeng.format() }
+}
+
+
+
+@TemplateModelHelpers
+val egenerklaeringPleieOgOmsorgsarbeidManuell = createAttachment<LangBokmalNynorskEnglish, OmsorgEgenManuellDto>(
+    title = newText(
+        Bokmal to "Egenerklæring om pleie- og omsorgsarbeid",
+        Nynorsk to "Eigenmelding om pleie- og omsorgsarbeid",
+        English to "Personal declaration that nursing and care work has been provided",
+    ),
+    includeSakspart = true
+) {
+    vedlegg(pesysData.returadresse) { saksbehandlerValg.aarEgenerklaringOmsorgspoeng.format() }
+}
+
+private fun OutlineOnlyScope<LangBokmalNynorskEnglish, *>.vedlegg(returadresse: Expression<ReturAdresse>, aarEgenerklaering: (TextScope<*,*>) -> Expression<String>) {
```

Comment:
Hva er grunnen til at `aarEgenerklaering` er funksjon som tar inn TextScope som argument, og ikke bare `Expression<String>`?

Reply:
Litt kompleksitet må vi jo ha for at det skal bli artig.

Eventuelt: eg innbilte meg at det var nødvendig, men det er det jo ikkje. Forenklar.